### PR TITLE
Make umich-lib-ui components local; package updates

### PR DESCRIPTION
**Pull request by jonearley**
## What's new?

This PR brings the React-based design system that supports this project locally at the `@reusable` alias. This makes things like package management and security updates easier and colocates the systems. Since the Design System is no longer supporting React distributions, these concerns are best left in this project, where they are solely used.

What are the main changes in this PR?

- Migrate [React design system components (mlibrary/umich-lib-ui)](https://github.com/mlibrary/umich-lib-ui) components locally to this project and aliased as `@reusable`.
- Security and npm package updates.
- Display each event occurrence -- LIBWEB-792 #148

So 99% of this PR is copied code from the `umich-lib-ui` repo and adjustments to import statements.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,13 +4,14 @@
...
+    "@reach/alert": "^0.13.0",
     "@reach/combobox": "^0.8.5",
     "@reach/dialog": "^0.8.5",
     "@reach/visually-hidden": "^0.8.2",
     "@reach/window-size": "^0.10.5",
     "@umich-lib/components": "^1.0.4",
-    "@umich-lib/core": "^2.0.6",
-    "gatsby": "^2.24.84",
+    "gatsby": "^2.31.1",
+    "gatsby-alias-imports": "^1.0.6",
```
Comment (bridgetburke) [inline package.json:14]: This is a nice little package.

**Review by bridgetburke — APPROVED** [review]
I reviewed every file. I'm glad to see the movement to contain the components within reusable. Good call!

**Comment by erinesullivan** [comment]
@jonearley Even though I couldn't run the site on this branch, all of the files look good. The only question I have is why does `src/reusable/expandable.js` exist and is empty?

**Comment by jonearley** [comment]
@erinesullivan Thanks! I've removed that file.

**Review by vagabundeo — COMMENTED** [review]
dont merge til tomorrow. otherwise overdue

**Review by erinesullivan — APPROVED** [review]
The empty file has been removed. All of the files look good. I locally ran the site on the `dev-logging` branch, since this branch was merged in with it, and the site didn't have any problems.

Good job!

**Comment by jonearley** [comment]
Thank you! 💯

**Pull request merged**